### ISSUE TITLE
The difference between minimum and maximum values should be at least 4

### ISSUE THREAD

**Issue by ArturFormella** [dverite/permuteseq#1]
Hello!
Not sure if its a bug but it is worth to notice:
```
create sequence seq0
    MINVALUE -10000
    start -10000
    MAXVALUE 9223372036854775800
    CACHE 1;

SELECT permute_nextval('seq0'::regclass, 123456789012345 ) FROM generate_series(-10000,-7000);

```

> ERROR:  sequence too short to encrypt.
> HINT:  The difference between minimum and maximum values should be at least 4.

If MINVALUE = 0 everything is ok.

**Comment by dverite** [comment]
It's an integer overflow bug when checking the sequence's range. I've just committed a fix, thanks for the report!

**Comment by dverite** [comment]
Fixed by 11c1f86
